environment/src/test/java/com/sequenceiq/environment/environment/dto/AuthenticationDtoConverterTest.java

```diff
@@ -59,4 +67,19 @@ void authenticationToDto() {
                 .matches(m -> Objects.equals(m.getPublicKeyId(), environment.getPublicKeyId()))
                 .matches(m -> Objects.equals(m.isManagedKey(), environment.isManagedKey()));
     }
+
+    @Test
+    void testSshKeyCreation() {
```

Comment:
I would like to see more use case here based on the regex

Reply:
It's not the place to test it. It has it's tests in the PublicKeyValidatorTest

Reply:
👍